### ISSUE TITLE
Listings with no reviews shouldn't display stars on search results

### ISSUE THREAD

**Issue by rhcastilhos** [OpenBazaar/openbazaar-desktop#884]
0.0 stars means bad thing

![nostar](https://user-images.githubusercontent.com/10789865/30464451-f2988754-99a7-11e7-8896-a7074ef524db.png)

**Comment by morebrownies** [comment]
Agreed. After poking around a bit on alternative platforms, it does appear that items without ratings typically do not display the star.

I support removing the ⭐️  when a listings has 0 reviews.
